src/selectors/entities/preferences.js

```diff
@@ -98,7 +120,7 @@ const getThemePreference = createSelector(
     }
 );
 
-export const getTheme = createSelector(
+export const getTheme = createShallowSelector(
```

Comment:
This doesn't need a shallow selector since the only way for a preference to change right now is if it's value changes or if a team-specific theme is added/removed

Reply:
Well actually the whole thing changes whenever the websocket re-connects as it replaces the whole `myPreferences` object

Reply:
Ah fair enough